config/routes.rb

```diff
@@ -122,6 +122,8 @@
     get 'limited', to: 'example#limited', as: :limited
     get 'status', to: 'admin#status'
 
+    get 'eligibility', to: 'claimants#eligibility', as: :eligibility
```

Comment:
I think it might be best to wrap this in a feature flag for the time being (that way it can be turned on and off on the different environments) [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/Feature-toggles.1859780873.html#Featuretoggles-Addinganewfeaturetoggle(backend)) and [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/Using-feature-toggles-to-progressively-release-changes.590938137.html) is the documentation for that.

```suggestion
if Flipper.enabled?(:feature_flag)
  get 'eligibility', to: 'claimants#eligibility', as: :eligibility
end
```